### PR TITLE
Views: Standardize Color Tokens

### PR DESCRIPTION
**Pull request by gblakeman**
Sets up some [proposed base-color tokens](https://www.figma.com/file/h04z1oXw4D5P2TyX8gRjCE/Plugin-UI-Library?node-id=705%3A18170) as a starting point for all of our plugins and simplifies the tokens currently in-use with _Realish_. `rgba` style colors are now tokenized as well.

A few notes:

* I am only using base tokens in any of the `Figma*` components since I can foresee us either moving them to our shared tools or even deprecating them if they’re not used in any of our newer designs.
* There may be a few base tokens currently not in use, but they should be utilized in the next round of work. Otherwise, I’ll remove them.

Plugin Base Colors

### CODE REVIEW

**Review by joeyyang — COMMENTED** [review]
Looks good so far! A few files where it looks like we've swapped semantic tokens for the new base tokens rather than the new semantic tokens - wanted to confirm with you that's what you meant to do.

File: src/views/forms-controls/FigmaSelectMenu.svelte [hunk 1]
```diff
@@ -325,11 +325,11 @@
     cursor: default;
     border: 1px solid transparent;
     border-radius: 2px;
-    background-color: $wl-ctrl-secondary;
+    background-color: $black-0o;
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/views/forms-controls/FigmaSelectMenu.svelte:328]: See my first bullet note on `Figma*` components.

File: src/views/forms-controls/ButtonIconLegacy.svelte [hunk 2]
```diff
@@ -34,7 +34,7 @@
       height: em(12);
 
       :global(svg path) {
-        fill: $wl-action-tertiary;
+        fill: $black-40o;
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/views/forms-controls/ButtonIconLegacy.svelte:37]: Yeah, this is a legacy button. It’ll go away once I complete my 2.0 buildout and I didn’t want to define a new semantic token for it.

File: src/views/Item.svelte [hunk 3]
```diff
@@ -116,10 +116,10 @@
       width: em($size);
       height: em($size);
       border-radius: em(math.div($size, 6));
-      background-color: $wl-action-tertiary;
-      background-image: linear-gradient(0deg, $wl-accent-tertiary 0%, $wl-action-tertiary 100%);
+      background-color: $black-40o;
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/views/Item.svelte:119]: Sorry, should’ve given you better notes. Like `ButtonIconLegacy`, this component changes in the next round of work and I didn’t want to define a temporary semantic token.

File: src/assets/css/shared/_colors.scss [hunk 4]
```diff
@@ -1,47 +1,69 @@
...
+$wl-canvas-secondary: $black-15o;
+$wl-canvas-tertiary: $blue-10o;
 
-$wl-type-01: $black;
-$wl-type-01-invert: $white;
-$wl-type-02: $grey02;
-$wl-type-03: $red01;
-$wl-type-04: $purple01;
-$wl-type-05: $blue01;
+// -- type
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/assets/css/shared/_colors.scss:37]: Typography – I’ll clarify the comment.

File: src/assets/css/shared/_colors.scss [hunk 5]
```diff
@@ -1,47 +1,69 @@
...
-$grey02: #979797;
-$grey03: #dfdfdf;
-$grey04: #f0f0f0;
-$blue01: #18a0fb;
-$blue02: #e8f6ff;
-$purple01: #7b61ff;
-$purple02: #f5f3ff;
-$red01: #f24822;
+// ----- base colors
+$black-100o: #000;
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/assets/css/shared/_colors.scss:4]: Longer works for me. I used to care a lot about formatting of colors that just isn’t necessary anymore considering how browser dev tools will do what they want anyway… 😂

**Review by joeyyang — APPROVED** [review]
Looks good this is awesome!

**Pull request merged**